src/_includes/hubspot_cta.liquid

```diff
@@ -0,0 +1,19 @@
+{%- if site.lang == 'es' -%}
+  <span class="{{ include.class }}" id="hs-cta-wrapper-{{ include.hubspot-id }}">
+    <span class="hs-cta-node hs-cta-{{ include.hubspot-id }}" id="hs-cta-{{ include.hubspot-id }}">
+      <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
+      <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/{{ include.hubspot-id }}" ></a>
+    </span>
+    <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, '{{ include.hubspot-id }}', {}); </script>
+  </span>
+{%- else -%}
+  <span class="{{ include.class }}" id="hs-cta-wrapper-{{ include.hubspot-id }}">
+    <span class="hs-cta-node hs-cta-{{ include.hubspot-id }}" id="hs-cta-{{ include.hubspot-id }}">
```

Comment:
Are these classes really needed if we're doing the CSS ourselves?

Reply:
I've taken `hs-cta-node` out, but the class with the id has to stay otherwise the cta isn't loaded in.

Amended in this commit: [8ee9826](https://github.com/codurance/site/commit/8ee98269d1eeee338bda3fa4659dda2da208b8e8)